owl/SOMA.owl

```diff
@@ -599,6 +599,17 @@ Sub-properties are used to distinct between different cases of event endpoint re
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#isRecordedBy -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/SOMA.owl#isRecordedBy">
```

Comment:
what about **includesInformation** as a property to link an Episode to InformationObject's instead ?
Then the description of the situation could state that it can only be satisfied by Episodes that include certain types of information objects.

I do not like the recorded property much.
I would rather say something like `isRecordedBy MonitoringDevice` or `MonitoringFacility`.
Actually, explicating MonitoringFacility would allow to provide more meta information in the future about the actual hardware and software employed for the monitoring, and e.g. data frequency, format., etc

Reply:
Sure, we could have includesInformation instead. I used records because records can be used between the event and the recording, as in "our journal records our activities".

Reply:
Actually, includesInformation sounds a bit too vague, I'll think of something else.